deployments/helm-chart/values.yaml

```diff
@@ -185,9 +185,6 @@ controller:
   ## Enable OIDC policies.
   enableOIDC: false
 
-  ## Include year in log header. This parameter will be removed in release 2.7 and the year will be included by default.
-  includeYear: false
-
```

Comment:
This shouldn't be removed

Reply:
@lucacome 
Please take a look at [this](https://github.com/nginxinc/helm-charts/blob/master/stable/nginx-ingress-0.14.0.tgz) (and later) version of the helm chart. 
The `values.yaml` doesn't contain `includeYear` and no one of the templates doesn't use this parameter. 
If you'll pass this parameter to `args` the application will raise an error. Why? Because this flag was removed.
So, I don't see any reason to keep this parameter.
You can try it on your own.

Reply:
@0m1xa the flag wasn't removed, there hasn't been a version release with it yet. That's why I suggested you try with `edge` which is a container built from the `main` branch instead of a release. Or you can build the container yourself from your branch and see that it works with `includeYear`.

Reply:
@0m1xa as @lucacome mentioned, there is not a version released yet with that specific argument. It will be in a future release which is why it needs to remain in the code.
`edge` should work or building the container from your branch are options.

Reply:
Ok. I've reverted this